domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -100,6 +99,88 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+  /** Marks full story progress for a particular profile. */
+  fun markFullProgressForAllTopics(profileId: ProfileId, timestampOlderThanAWeek: Boolean) {
```

Comment:
Suggest: ``timestampOlderThanOneWeek``.

Also, please add a ``@param`` to the KDoc explaining what this field means to provide context into how to specify it.

Reply:
Done